src/main/java/com/parksupark/soomjae/server/member/dto/MemberResponse.java

```diff
@@ -1,21 +1,10 @@
 package com.parksupark.soomjae.server.member.dto;
 
-import com.parksupark.soomjae.server.member.entity.Member;
-import lombok.Getter;
+import lombok.Data;
 
-@Getter
+@Data
```

Comment:
p5; 

취향문제 일수도 있을 것 같은데 Setter가 포함되어 있어 저는 선호하지 않습니다!
혹은 setter만 접근 레벨을 설정해주는 방법도 있습니다.

```suggestion
@Data
@Setter(AccessLevel.NONE)
```

Reply:
이전에 한번 Data를 써볼까 생각했다가 현재는 저도 그냥 안쓰려고 하는데 제가 빼먹은것 같습니다 허허